### DISCUSSION TITLE
Rego policy - ignore multiple vulnerabilities by CVE ID

### DISCUSSION THREAD

**Discussion by imbelchev** [aquasecurity/trivy#4656] [Q&A] [answered]
### Question

Hi there,
I have a question regarding the Rego policy.

I would like to suppress multiple vulnerabilities using the CVE identifier under single `ignore{}` section.
I am only able to ignore multiple CVEs by having separate `ignore{}`-s with single CVE ID specified inside.

Is it possible to achieve similar behaviour like with the [CWE IDs](https://github.com/aquasecurity/trivy/blob/9a279fa7bb5ccdcda642f99ac2dfd80551082ee2/contrib/example_policy/advanced.rego#LL89C1-L89C1)?

Thanks!

---

Current:
```
package trivy

ignore {
	input.VulnerabilityID == "CVE-2022-43552"
}

ignore {
	input.VulnerabilityID == "CVE-2023-23916"
}
```

Desired:
```
package trivy

ignore {
	deny_vulnerability_ids := {
		"CVE-2022-43552",
		"CVE-2023-23916"
	}
	count({x | x := input.VulnerabilityIDs[_]; x == deny_vulnerability_ids[_]}) == 0
}
```

**Comment by knqyf263** [comment, marked as answer]
Did you try the following?
```
input.VulnerabilityID == deny_vulnerability_ids[_]
```

**Reply by imbelchev** [reply]
Wow, this is crazy... Thank you!
